email.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+  <head>
+	    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	    <title>FrontlineCloud</title>
+	    <link rel="stylesheet" type="text/css" href="assets/css/normalize.css">
+	    <link rel="stylesheet" type="text/css" href="assets/css/main.css">
+  </head>
+
+  <body>
+
+		<div id="wrapper">
+
+			<div id="header">
+				<img src="https://gallery.mailchimp.com/508a0aa7c5864b754935dabed/images/9c2db43a-d929-45d9-a8ac-2d1b0561e588.png" alt="FrontlineCloud"/>
+			</div>
+
+			<div id="content-body">
+				<p>Hello,</p>
+				<p>Someone - hopefully you - has requested that we reset the password to your FrontlineCloud account.</p>
+				<p>
+					If you made the request, you will be able to 
+					<a href="https://cloud.frontlinesms.com/register/resetPassword?t=7896479904cf4f2eae6370b81754e860" target="_blank">
+					set a new password here.https://cloud.frontlinesms.com:443/register/resetPassword?t=7896479904cf4f2eae6370b81754e860
+					</a>
+				</p>
+				<p>
+					If this request was made in error, please contact our support team at 
+					<a href="mailto:support@frontlinesms.com" target="_blank">support@frontlinesms.com.</a>
+					We will respond to you as quickly as possible.
+				</p>
+				<p>
+					Thank You!<br>
+					The FrontlineCloud Team<br>
+					<a href="http://www.frontlinesms.com/" target="_blank">http://frontlinesms.com</a>
+				</p>
+			</div>
+
+			<div id="footer">
+				<p>

```

Comment:
There is incorrect use of the <br> tag here. br elements must not be used for separating thematic groups in a paragraph. http://www.w3.org/html/wg/drafts/html/master/semantics.html#the-br-element